File: 프로그래머스/1/12917. 문자열 내림차순으로 배치하기/문자열 내림차순으로 배치하기.py
def solution(s):
    answer = ''
    low_str = []
    up_str = []
    for str in s:
        if str.isupper():
            up_str.append(str)
        else:
            low_str.append(str)
    low_str.sort(reverse=True)
    up_str.sort(reverse=True)
    answer += "".join(low_str)
    answer += "".join(up_str)
    return answer

File: 프로그래머스/1/12917. 문자열 내림차순으로 배치하기/문자열 내림차순으로 배치하기.swift
func solution(_ s:String) -> String {
    var lower = ""
    var upper = ""
    for c in s {
        if c.isLowercase { lower.append(c) }
        else { upper.append(c) }
    }
    var answer = ""
    answer += String(lower.sorted(by:>))
    answer += String(upper.sorted(by:>))
    return answer
}